src/crimson/osd/scheduler/mclock_scheduler.cc

```diff
@@ -21,45 +21,129 @@
 
 namespace dmc = crimson::dmclock;
 using namespace std::placeholders;
+using namespace std::literals;
 
 #define dout_context cct
-#define dout_subsys ceph_subsys_osd
```

Comment:
there's a lot of duplication from classic in this file - is it worth abstracting out key differences so the large common pieces can be shared? e.g. default profiles and config information?

Reply:
done